iceoryx_utils/source/posix_wrapper/shared_memory_object.cpp

```diff
@@ -20,18 +20,45 @@
 #include "iceoryx_utils/cxx/smart_c.hpp"
 #include "iceoryx_utils/platform/fcntl.hpp"
 #include "iceoryx_utils/platform/unistd.hpp"
+#include "iceoryx_utils/posix_wrapper/signal_handler.hpp"
 
 #include <bitset>
 #include <cstdlib>
 #include <cstring>
 #include <iostream>
+#include <mutex>
 
 namespace iox
 {
 namespace posix
 {
 constexpr void* SharedMemoryObject::NO_ADDRESS_HINT;
 
+static struct
+{
+    SharedMemory::Name_t name;
+    uint64_t memorySizeInBytes;
+    AccessMode accessMode;
+    OwnerShip ownerShip;
+    const void* baseAddressHint;
+    mode_t permissions;
+    std::mutex sigbusHandlerMutex;
+} memsetSigbusDebugInfo;
+
+static void memsetSigbusHandler(int)
+{
+    std::cerr
```

Comment:
`std::cerr` is not allowed in a signal handler, neither is `exit`. You have to use `write` and `_exit`. @FerdinandSpitzschnueffler has some experience with this

Reply:
@elBoberido it is not like I do not believe you but do you have some kind of source for this requirement? I am looking all over the place and see that we do this but I cannot find some kind of man page where this is stated!